.github/workflows/ci.yml

```diff
@@ -12,11 +12,19 @@ jobs:
   build-test:
     runs-on: ${{ matrix.os }}
     strategy:
-      matrix:
-        # Python 3.8 is the last non-EOL version. Also adapt tox.ini
-        python-version: ['3.8', 'pypy3.10', '3.12']
-        os: [ubuntu-latest, windows-latest]
       fail-fast: false
+      matrix:
+        # Python 3.9 is the last non-EOL version. Also adapt tox.ini
+        python-version: ['3.9', 'pypy3.11', '3.13']
+        # macOS on ARM, Ubuntu on x86, Windows oo X86
+        os: [macos-latest, ubuntu-latest, windows-latest]
+        include:
+          - python-version: 3.x
+            os: macos-13  # macOS on Intel
```

Comment:
I think the only platform difference that could have a difference to need standalone testing for macOS is ARM, so I don't get this. In fact, I don't get why we need any of the "include" things instead of just replacing 3.13 with 3.x.

Reply:
Done.